spring-boot/admin/src/main/resources/templates/admin/fragments/proxies/list.html

```diff
@@ -143,7 +143,7 @@ <h2 th:text="#{admin.proxies.list.title}">[Proxies Administration]</h2>
                                     <td class="text-center "><a th:href="|@{/admin/proxies/}${proxy.id}|"
                                                                 class="fa fa-bar-chart"/>
                                     </td>
-                                    <td th:text="${proxy.timeOfLastSuccessfulCallToAdmin != null ? #dates.format(new java.util.Date(proxy?.timeOfLastSuccessfulCallToAdmin)) : 'No sign of interaction'}"></td>
+                                    <td th:text="${proxy.timeOfLastSuccessfulCallToAdmin != null ? #dates.format(new java.util.Date(proxy?.timeOfLastSuccessfulCallToAdmin)) : #{admin.dashboard.gatewayhealthmonitor.notconnectedgateway}"></td>
```

Comment:
You added an opening `{` without a closing `}`.

Reply:
You are right, and after we still need to test if the #{} works inside of a elvis operator condition

Reply:
Done